Fluid/Fluid.csproj

```diff
@@ -10,21 +10,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Parlot" Version="0.0.5" />
+    <PackageReference Include="Parlot" Version="0.0.6" />
     <PackageReference Include="Microsoft.Extensions.FileProviders.Abstractions" Version="1.1.1" />
     <PackageReference Include="TimeZoneConverter" Version="3.3.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.1'">
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
   </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp3.1'">
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)'=='net5.0'">
```

Comment:
Why we remove other target frameworks?!!

Reply:
I couldn't give why I added them. Can you think of any reason to keep them?

Reply:
May be you added them when it's `netcoreapp` not `netstandard`

Reply:
I think that since it's targeting netstd20 and netstd21, __and__ there is no specific dependency that could go away with netcoreapp, then it's not required to target netcoreapp.

Reply:
is the PR functional ready, I'd like to test few cases

Reply:
You should pay more attention to what I wrote, because I was looking at the csproj and realized that the netcoreapp targets actually remove some dependencies, so I am adding them back! With a comment this time ;)

Reply:
It's ready, I will push a v1 with the current master, create a release/1.x branch to follow up, and then start v2 by merging this one on master. At the same time migrate to "main"